src/store/participantsStore.js

```diff
@@ -526,6 +597,9 @@ const actions = {
 			sessionId: participantIdentifier.sessionId,
 			flags: PARTICIPANT.CALL_FLAG.DISCONNECTED,
 		})
+
+		// re-fetch conversation after leaving to update callFlag
+		await dispatch('fetchConversation', { token })
```

Comment:
The `inCall` flag is already set to disconnected just before this. Is the additional request needed?

Reply:
As long as we're keeping the elapsed time while the call is on, we need to distinguish `call_ended` and `call_left`. If design team is up for changing the behaviour, it'll simplify the code